### PR TITLE
Misc. typos

### PR DESCRIPTION
**Pull request by luzpaz**
Found via `codespell` and `grep`

### CODE REVIEW

**Comment by trelau** [comment]
@luzpaz thanks for the effort. I noticed that in some places the method name was fixed like [here](https://github.com/LaughlinResearch/pyOCCT/pull/3/commits/1fa951c868fe951891bf766cdd90df08e8468f6f#diff-1056cdf8de3d008c24e7c6d4cb59e08fR918). For now, I think it would be best to match the C++ name, even if it's misspelled. Is there a way to tell the tools to not correct the method names?

**Comment by trelau** [comment]
There are some others at first glance [see here](https://github.com/LaughlinResearch/pyOCCT/pull/3/commits/1fa951c868fe951891bf766cdd90df08e8468f6f#diff-53407ba66bd9b1ecdc2dbdd4029b7386L213). I didn't know if there was a way to run the spell checker without it modifying the method names. Otherwise, if we want to incorporate this we'll need to manually review the changes carefully and make sure no method names were changed. I did a quick scroll through the GitHub commit, but not a deep dive.

**Comment by trelau** [comment]
One other thing: I remember seeing a few cases were the spell checker fixed some argument names. This might be ok, but if a user ever tried to use keyword arguments and match the OCCT spelling, it would throw an error. Perhaps it's best to just target corrections in the docstrings rather than method and argument names? Is that possible?

**Comment by looooo** [comment]
> Perhaps it's best to just target corrections in the docstrings rather than method and argument names? Is that possible?  

maybe running the spell checker directly on occt?

**Comment by trelau** [comment]
I agree with @looooo suggestion of running it on OCCT, and then (hopefully) getting OpenCASCADE to incorporate it into the project and future releases. This would be the best long-term solution in my opinion.

**Comment by luzpaz** [comment]
If you look at their recent releases, I have submitted typo fixes to smesh, geom, kernel, medcoupling, netgensomething.

**Comment by trelau** [comment]
Those are Salome Platform modules, right? Have you submitted anything to the OpenCASCADE SAS [repo](https://dev.opencascade.org/index.php?q=home/resources)?

Either way, I'm ok with pulling in fixes in the docstrings, but not yet on anything that would change a method or argument name and therefore not match the C++ source code. If you can revert any of these changes then I am happy to incorporate the request.

If you are able to get fixes incorporated into OpenCASCADE, that would be the best solution as it would propagate to any downstream projects that are using OCC.

**Comment by luzpaz** [comment]
No worries about changing source strings. I'm submitting stuff upstream and when the time comes they will trickle downstream. I think I may have purged the source string relevant stuff out of the PR. The issue is that the way the strings are written they aren't easy to review.

**Comment by trelau** [comment]
If I understand you correctly, are you suggesting we close this PR since your changes will be applied upstream and eventually make it here?

File: src/AdvApp2Var.cpp [hunk 1]
```diff
@@ -268,7 +268,7 @@ PYBIND11_MODULE(AdvApp2Var, mod) {
...
 	cls_AdvApp2Var_ApproxAFunc2Var.def("VFrontError", (Standard_Real (AdvApp2Var_ApproxAFunc2Var::*)(const Standard_Integer, const Standard_Integer) const ) &AdvApp2Var_ApproxAFunc2Var::VFrontError, "returns the error max of the BSplineSurface of range Index on a VFrontier", py::arg("Dimension"), py::arg("Index"));
 	cls_AdvApp2Var_ApproxAFunc2Var.def("CritError", (Standard_Real (AdvApp2Var_ApproxAFunc2Var::*)(const Standard_Integer, const Standard_Integer) const ) &AdvApp2Var_ApproxAFunc2Var::CritError, "None", py::arg("Dimension"), py::arg("Index"));
-	// FIXME cls_AdvApp2Var_ApproxAFunc2Var.def("Dump", (void (AdvApp2Var_ApproxAFunc2Var::*)(Standard_OStream &) const ) &AdvApp2Var_ApproxAFunc2Var::Dump, "Prints on the stream o informations on the current state of the object.", py::arg("o"));
+	// FIXME cls_AdvApp2Var_ApproxAFunc2Var.def("Dump", (void (AdvApp2Var_ApproxAFunc2Var::*)(Standard_OStream &) const ) &AdvApp2Var_ApproxAFunc2Var::Dump, "Prints on the stream o information on the current state of the object.", py::arg("o"));
```
Comment (luzpaz) [inline src/AdvApp2Var.cpp]: `Prints on the stream o information on current state of the object` This should be re-written

**Comment by luzpaz** [comment]
> If I understand you correctly, are you suggesting we close this PR since your changes will be applied upstream and eventually make it here?

@trelau not necessarily, but that prompts a question, are all the text string descriptions from upstream or did you tweek them for the purposes of this project?

BTW, it's easier to read changes using split view:
https://github.com/LaughlinResearch/pyOCCT/pull/3/files?diff=split

**Comment by trelau** [comment]
The text strings are all from upstream and taken from the source header files if available. The only change to anything text related was adding a trailing underscore to the names of static functions.

**Comment by luzpaz** [comment]
I'll close this PR then. Quick question, what's the difference between Salome and OpenCASCADE SAS  ?

**Comment by trelau** [comment]
I think Salome is an application developed (in part) by OpenCASCADE the company? It uses OpenCASCADE geometry kernel under the hood, but has it's own layers for geometry, meshing, etc.

**Comment by trelau** [comment]
Closing as discussed. Typo corrections will be applied upstream.

**Pull request closed without merging**